### PR TITLE
#2 Add Ansible on Azure docs

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Resolves issue #2  Add Ansible on Azure docs to `/titles/aap-on-azure/`

- Add directories and files for modules, assemblies, titles, common content, images, attributes
- Add symlinks (see `tree` output below)

```
➜  /titles/aap-on-azure/
$ tree .
.
├── aap-common -> ../../aap-common
├── aap-on-azure -> ../../assemblies/aap-on-azure
├── aap-on-azure.asciidoc
├── attributes -> ../../attributes
├── images -> ../../images
├── master.adoc
└── stories.adoc


➜  /aap-docs/assemblies/
$ tree .
.
└── aap-on-azure
    ├── aap-on-azure -> ../../modules/aap-on-azure
    ├── assembly-aap-azure-connecting-to-aap.adoc
    ├── assembly-aap-azure-direct-peering.adoc
    ├── assembly-aap-azure-hub-spoke-peering.adoc
    ├── assembly-aap-azure-install.adoc
    ├── assembly-aap-azure-intro.adoc
    ├── assembly-aap-azure-network-peering.adoc
    └── assembly-aap-azure-vwan-peering.adoc
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@ html/
 .sw*
 .*.sw*
 master.html
+aap-on-azure.html
```
Comment (oraNod) [inline .gitignore:9]:
```suggestion
*.html
```

**Review by oraNod — CHANGES_REQUESTED** [review]
One minor thing in this review. I'd like to push a commit and then talk through the changes. Squashing this to a single commit would make it easier to compare.

Also please create an issue and remove all reference to the Jira ticket, including the commit message. Reference the issue number that you create in the new commit message with `#<issue-number>` That way the PR automatically links to the issue.

**Comment by ariordan-redhat** [comment]
Created issue 2. Commits squashed.

**Comment by ariordan-redhat** [comment]
@oraNod Your updates look good to me - thanks.

**Review by oraNod — APPROVED** [review]
LGTM

**Comment by oraNod** [comment]
Thanks @ariordan-redhat

**Pull request merged**